### PR TITLE
In the commify answer, drop Number::Format for CLDR::Number

### PR DESCRIPTION
**Pull request by briandfoy**
For #88

### CODE REVIEW

**Review by davorg — APPROVED** [review]

**Comment by briandfoy** [comment]
What needs to happen for this to move forward?

**Comment by briandfoy** [comment]
This PR has been approved. What needs to happen for it to be merged?

**Comment by briandfoy** [comment]
I'm dropping the patch file here for when I delete my fork later this month. If a PR isn't going to go forward, I need it out of my queue of open things.

[0001-In-the-commify-answer-drop-Number-Format-for-CLDR-Nu.patch](https://github.com/perl-doc-cats/perlfaq/files/12862134/0001-In-the-commify-answer-drop-Number-Format-for-CLDR-Nu.patch)

File: lib/perlfaq5.pod [hunk 1]
```diff
@@ -704,20 +704,46 @@ X<number, commify>
...
+rules. The bare C<en> uses commas between groups of three, the C<de>
+uses dots and groups of three, and the C<en-IN> uses the 3:2:2 rule with
+commas:
+
+    en-US 1,234,567,890
+    de-DE 1.234.567.890
+    en-IN 1,23,45,67,890
+
+L<CLDR::Number> has many more advanced features to handle just about
+anything that you might want. Olaf Anders article "Stop Writing Your Own
```
Comment (karenetheridge) [inline lib/perlfaq5.pod:731]:
```suggestion
anything that you might want. Olaf Alders' article "Stop Writing Your Own
```

**Review by karenetheridge — CHANGES_REQUESTED** [review]

**Pull request merged**